cli/command/image/build/context.go

```diff
@@ -242,9 +242,9 @@ func getWithStatusError(url string) (resp *http.Response, err error) {
 	body, err := ioutil.ReadAll(resp.Body)
 	resp.Body.Close()
 	if err != nil {
-		return nil, errors.Wrapf(err, msg+": error reading body")
+		return nil, errors.Wrap(err, fmt.Sprintf("%s: error reading body", msg))
```

Comment:
oh, perhaps I explained badly; this should either be;

```go
return nil, errors.Wrap(err, msg+": error reading body")
```

or

```go
return nil, errors.Wrapf(err, "%s: error reading body", msg)
```

Reply:
Thank you!